src/SDK/SmartDetectorsExtensionsPackage/SmartDetectorsExtensionsPackage.nuproj

```diff
@@ -23,7 +23,7 @@
   <Import Project="$(NuProjPath)\NuProj.props" Condition="Exists('$(NuProjPath)\NuProj.props')" />
   <PropertyGroup Label="Configuration">
     <Id>Microsoft.Azure.Monitoring.SmartDetectors.Extensions</Id>
-    <Version>1.0.12.4</Version>
+    <Version>1.0.12.5</Version>
```

Comment:
that's not needed

Reply:
How come? If the SmartDetectorsPackage is update, the extension (which references it) should also be updated, no?

---
In reply to: [275232950](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/59#discussion_r275232950) [](ancestors = 275232950)

Reply:
nope - as long as it's not breaking change then there's no need

and since the extensions pakcage doesn't use the tracer at all, we should be good

---
In reply to: [275250287](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/59#discussion_r275250287) [](ancestors = 275250287,275232950)